opentelemetry-zipkin/Cargo.toml

```diff
@@ -38,6 +38,7 @@ http = "0.2"
 reqwest = { version = "0.11", optional = true, default-features = false }
 surf = { version = "2.0", optional = true, default-features = false }
 thiserror = { version = "1.0"}
+futures = "0.3"
```

Comment:
Can we save on dependencies by using futures-util (or even futures-core) for most of these?

Reply:
With the exception of the current Jaeger exporter implementation, we should be able to get away with using only futures-util. I'll make that change.

Reply:
Ah, and maybe in some cases even futures-core (at least for Zipkin)

Reply:
Done in the latest commit.